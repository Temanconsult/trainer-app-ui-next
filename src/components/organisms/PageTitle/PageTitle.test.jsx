import React from 'react'
import { render } from '@testing-library/react'
import PageTitle from '.'

jest.mock('next/head', () => ({
  __esModule: true,
  // eslint-disable-next-line
  default: ({ children }) => <>{children}</>
}))

describe('PageTitle', () => {
  it('creates a <head> element suitable for global usage across all pages', () => {
    const mockTitle = 'mock title for test'
    const { asFragment, getByTestId } = render(<PageTitle title={mockTitle} />)

    expect(getByTestId('title').textContent).toBe(mockTitle)
    expect(asFragment()).toMatchSnapshot()
  })
})
