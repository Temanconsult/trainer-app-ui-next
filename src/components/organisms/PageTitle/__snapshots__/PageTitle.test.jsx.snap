// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle creates a <head> element suitable for global usage across all pages 1`] = `
<DocumentFragment>
  <title
    data-testid="title"
  >
    mock title for test
  </title>
</DocumentFragment>
`;
